code/modules/shuttle/bluespace_shuttle_pod/pod_computer.dm

```diff
@@ -20,6 +20,9 @@
 	shuttle_creator.reset_saved_area(FALSE)
 
 /obj/machinery/computer/shuttle_flight/custom_shuttle/bluespace_pod/ui_interact(mob/user, datum/tgui/ui)
+	if(isobserver(user))
+		user.visible_message("<span class ='warning'>Ghosts can't rename the shuttle!.<span>")
+		return
```

Comment:
This will prevent them from using the console entirely. Add !isobserver(user) check to the next check.

Reply:
> prevent them from using the console entirely.

prevent whom? The player in game?

![image](https://user-images.githubusercontent.com/62388554/205722619-7cf0d0e0-401f-4b19-9500-e4e5ba19fc53.png)
looks like it works fine

Reply:
Ohhh, you mean you want ghosts to still be able to see the console. Alright

Reply:
addressed 
![image](https://user-images.githubusercontent.com/62388554/205725645-d11fda45-218e-4da8-befe-f267f3f668c0.png)